sync_issues_to_jira/sync_issue.py

```diff
@@ -458,4 +457,21 @@ def _get_jira_comment_body(gh_comment, body=None):
 
 def _get_jira_label(gh_label):
     """ Reformat a github API label item as something suitable for JIRA """
+    # ignore status, resolution and platform: windows labels
+    if _check_issue_label(gh_label["name"]) is None:
+        return
```

Comment:
At line 257 the GitHub labels get reformatted into JIRA labels like this:
```python
"labels": [_get_jira_label(l) for l in gh_issue["labels"]],
```
... returning None here will cause some None values here, possible these are ignored but I think they might cause errors instead.